Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -282,6 +282,7 @@ instance coeToLaurentSeries : Coe (RatFunc F) (LaurentSeries F) :=
 theorem coe_def : (f : LaurentSeries F) = coeAlgHom F f :=
   rfl
 
+attribute [-instance] RatFunc.instCoePolynomial in
```

Comment:
Oh, and we should leave a note about the diamond here and below.

Reply:
I included something in PR preamble and the Zulip link, is this ok? What do you mean with "below"?

Reply:
I *think* "below" refers to `coe_coe`, but we surely want a comment in the code explaining why we are removing the instance.

Reply:
Added first comment, no idea what to write at `coe_coe`.

Reply:
The same comment at `coe_coe` works fine.

Reply:
Done, thanks.